crates/forge/bin/cmd/test/mod.rs

```diff
@@ -898,10 +882,6 @@ impl Provider for TestArgs {
             dict.insert("show_progress".to_string(), true.into());
         }
 
-        if let Some(threads) = self.threads {
-            dict.insert("threads".to_string(), threads.into());
```

Comment:
we have `threads` in config but I don't really know where it should spawn the thread pool, and also it would conflict with the CLI arg if both are present